src/app/test-results/test-results-list/test-results-detail-list.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ResponseTestResultDetail, TestResultService } from 'api-client';
+
+@Component({
+  selector: 'wh-test-results-detail-list',
+  templateUrl: './test-results-detail-list.component.html',
+})
+export class TestResultsDetailListComponent implements OnInit {
+  @Input() buildId: number;
+  @Input() summaryId: number;
+  details: ResponseTestResultDetail[];
+  failedDetails: ResponseTestResultDetail[];
+  private fetched: boolean;
+
+  constructor(
+    private testResultService: TestResultService) { }
+
+  ngOnInit(): void {
+    if (!this.fetched) {
+      this.testResultService
+        .getBuildTestResultDetailList(this.buildId, this.summaryId)
+        .subscribe(details => {
+          this.details = details.list;
+          this.details.forEach(d => d.message = this.unescapeHtml(d.message));
```

Comment:
This HTML entities escaping really seems like a backend bug and nothing the frontend should deal with. Maybe an artifact of the XML parser?

I opt for removing this unescaping and file an issue on the wharf-api repo.

Reply:
The files themselves seem to contain it. I think XML, as HTML, generally does that to avoid issues.
Filing an issue on the wharf-api repo 👍🏻

Reply:
Issue created: https://github.com/iver-wharf/wharf-api/issues/133

Reply:
Would prefer keeping it, as some messages can look very off when escaped.
Then we can remove it once wharf-api unescapes for us.